README.md

```diff
@@ -158,23 +160,117 @@ is different. See each providers `help` command for the available actions.
 Note, when uploading to some cloud providers (e.g. gce) the image may need to be packaged
 with a different --format (e.g. --format=gce) when running `image_to_vm.sh`
 
-## plume
+### plume
 Plume is the Container Linux release utility. Releases are done in two stages,
 each with their own command: pre-release and release. Both of these commands are idempotent.
 
-### plume pre-release
+#### plume pre-release
 The pre-release command does as much of the release process as possible without making anything public.
 This includes uploading images to cloud providers (except those like gce which don't allow us to upload
 images without making them public).
 
-## plume release
+### plume release
 Publish a new Container Linux release. This makes the images uploaded by pre-release public and uploads
 images that pre-release could not. It copies the release artifacts to public storage buckets and updates
 the directory index.
 
-### plume index
+#### plume index
 Generate and upload index.html objects to turn a Google Cloud Storage
 bucket into a publicly browsable file tree. Useful if you want something
 like Apache's directory index for your software download repository.
 Plume release handles this as well, so it does not need to be run as part of
 the release process.
+
+## Platform Credentials
+Each platform reads the credentials it uses from different files. The `aws`, `esx`, `oci` and `packet`
+platforms support selecting from multiple configured credentials, call "profiles". The examples below
+are for the "default" profile, but other profiles can be specified in the credentials files and selected
+via the `--<platform-name>-profile` flag:
+```
+kola spawn -p aws --aws-profile other_profile
+```
+
+### aws
+`aws` reads the `~/.aws/credentials` file used by Amazon's aws command-line tool.
+It can be created using the `aws` command:
+```
+$ aws configure
```

Comment:
for consistency with mentioning the `--aws-profile` flag above, let's toss in `<--profile other_profile>` here